sw/device/tests/closed_source/BUILD.bazel

```diff
@@ -174,21 +167,18 @@ opentitan_functest(
 # `MANUFACTURER_HOOKS_DIR=/path/to/test_hooks
 #     bazel test @manufacturer_test_hooks//:statically_hooked_opensource_test`
 ################################################################################
-opentitan_functest(
+opentitan_test(
     name = "statically_hooked_opensource_test",
     srcs = ["@//sw/device/tests:example_test_from_flash.c"],
     dv = dv_params(
-        rom = "//sw/device/silicon_creator/rom:rom_with_fake_keys",
+        rom = "@//sw/device/silicon_creator/rom:rom_with_fake_keys",
     ),
-    key_struct = get_key_structs_for_lc_state(
-        CONST.LCV.TEST_UNLOCKED0,
-        spx = None,
-    )[0],
-    targets = [
-        "cw310_rom_with_fake_keys",
-        "cw310_test_rom",
-        "dv",
-    ],
+    exec_env = {
+        "@//hw/top_earlgrey:fpga_cw310_rom_with_fake_keys": None,
+        "@//hw/top_earlgrey:fpga_cw310_test_rom": None,
+        "@//hw/top_earlgrey:sim_dv": None,
+    },
+    rsa_key = {"@//sw/device/silicon_creator/rom/keys/fake/rsa:test_private_key_0": "test_key_0"},
```

Comment:
`//rules/opentitan:defs.bzl` exports a function called `rsa_key_for_lc_state` that _could_ be used here.  We'll have to check the form of the label it returns and make sure it is prefixed with an `@` symbol.

Reply:
Fixed.